docs/frc/4-Autonomous/1-Overview.md

```diff
@@ -18,17 +18,17 @@ Auto routines will be programmed in the form of commands, and therefore should g
 
 Here are a few examples of really riveting, top-tier autonomous robot performances.
 
-![254 Barrage Reveal](img/254barrageReveal.gif) [Source](https://www.youtube.com/watch?v=aFZy8iibMD0)
+[254 Barrage Reveal (2014)](https://youtu.be/aFZy8iibMD0?t=19) (warning: loud audio for headphone users)
```

Comment:
For these you should try doing embedded iframes (go to the link and go to share)